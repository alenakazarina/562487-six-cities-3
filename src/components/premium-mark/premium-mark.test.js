import React from 'react';
import renderer from 'react-test-renderer';
import PremiumMark from './premium-mark';

describe(`PremiumMark`, () => {
  it(`should render place card's PremiumMark`, () => {
    const tree = renderer.create(
        <PremiumMark prefix={`place-card__`} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`should render property's PremiumMark`, () => {
    const tree = renderer.create(
        <PremiumMark prefix={ `property__`} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
