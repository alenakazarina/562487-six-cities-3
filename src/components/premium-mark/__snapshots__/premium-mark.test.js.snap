// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PremiumMark should render place card's PremiumMark 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;

exports[`PremiumMark should render property's PremiumMark 1`] = `
<div
  className="property__mark"
>
  <span>
    Premium
  </span>
</div>
`;
